boards/redboard_redv/README.md

```diff
@@ -0,0 +1,62 @@
+Redboard Red-V B RISC-V Board
+==================================
+
+- https://www.sparkfun.com/products/15594
+
+Arduino-compatible dev board for RISC-V clone of the Hifive1.
+
+Programming
+-----------
+
+Running `make flash` should load the kernel onto the board. You will need a
+relatively new (i.e. from git) version of OpenOCD.
```

Comment:
Is this still true or will the latest OpenOCD release work?

Reply:
removed